HISTORY.rst

```diff
@@ -7,6 +7,8 @@ History
 * ``attrs`` and dataclass structuring is now ~25% faster.
 * Fix an issue structuring bare ``typing.List`` s on Pythons lower than 3.9.
   (`#209 <https://github.com/python-attrs/cattrs/issues/209>`_)
+* Fix structuring bare generics on Pythons lower than 3.9.
```

Comment:
Maybe be more specific with `bare lower-case generics`

Reply:
The thing is, these aren't even generics on 3.7. So more like `non-parameterized containers like list and dict` I guess?

Reply:
Yep, will do!